.github/workflows/check.yml

```diff
@@ -1,11 +1,11 @@
-name: ci
+name: check
 
 on:
   push:
     branches:
       - master
     tags:
-      - v*
+      - "^v*"
```

Comment:
1) Is this necessary? I'm not publishing any other tags, especially tags starting with `v` like `vercel` or something
2) Is this tested on your fork? Last time we've merged not working stuff, here I'm afraid of `^` not being glob compliant

Reply:
@MichalLytek Right! Fixed